url-updater/src/main/java/com/devonfw/tools/ide/url/tool/python/PythonUrlUpdater.java

```diff
@@ -7,6 +7,9 @@
 import org.slf4j.LoggerFactory;
 
 import com.devonfw.tools.ide.json.JsonMapping;
+import com.devonfw.tools.ide.tool.python.PythonFile;
+import com.devonfw.tools.ide.tool.python.PythonJsonObject;
+import com.devonfw.tools.ide.tool.python.PythonRelease;
```

Comment:
Shouldn't we also move these files here?
They are IMHO only used by the url-updaters and not by IDEasy and URL-Metadata.

Reply:
Moved to url-updater.

Reply:
same here. Still in cli module.